examples/inference/benchmark_llama.py

```diff
@@ -110,7 +110,7 @@ def benchmark_inference(args):
                 max_output_len=args.output_len,
                 prefill_ratio=1.2,
             )
-            engine = InferenceEngine(model, tokenizer, inference_config, verbose=True)
+            engine = InferenceEngine(model, inference_config, tokenizer, verbose=True)
```

Comment:
Keep the order to `model, tokenizer, inference_config` would be better to me.

Reply:
changed